Failing test: Security Solution Cypress.x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints·cy·ts - Endpoints page should update endpoint policy on Endpoint should update endpoint policy on Endpoint

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 20000ms: expected '/app/security/administration/endpoints' to match /^\/app\/security\/administration\/policy\/[a-f0-9-]+\/settings$/
    at Context.eval (webpack:///./e2e/endpoint_list/endpoints.cy.ts:100:48)
```

First failure: [kibana-on-merge - 9.0](https://buildkite.com/elastic/kibana-on-merge/builds/66744#01963e87-e709-40ff-b2fa-6632f4340e08)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints·cy·ts","test.name":"Endpoints page should update endpoint policy on Endpoint should update endpoint policy on Endpoint","test.failCount":8}} -->

--- a/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/automated_response_actions/automated_response_actions.cy.ts
+++ b/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/automated_response_actions/automated_response_actions.cy.ts
@@ -19,8 +19,7 @@ import { createEndpointHost } from '../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../tasks/delete_all_endpoint_data';
 import { enableAllPolicyProtections } from '../../tasks/endpoint_policy';
 
-// Failing: See https://github.com/elastic/kibana/issues/207773
-describe.skip(
+describe(
   'Automated Response Actions',
   {
     tags: ['@ess', '@serverless'],

--- a/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts
+++ b/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts
@@ -28,8 +28,7 @@ import { createEndpointHost } from '../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../tasks/delete_all_endpoint_data';
 import { enableAllPolicyProtections } from '../../tasks/endpoint_policy';
 
-// Failing: See https://github.com/elastic/kibana/issues/218441
-describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
+describe('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
   let indexedPolicy: IndexedFleetEndpointPolicyResponse;
   let policy: PolicyData;
   let createdHost: CreateAndEnrollEndpointHostResponse;

--- a/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/file_operations.cy.ts
+++ b/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/file_operations.cy.ts
@@ -21,14 +21,12 @@ import { enableAllPolicyProtections } from '../../../tasks/endpoint_policy';
 import { createEndpointHost } from '../../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../../tasks/delete_all_endpoint_data';
 
-// Failing: See https://github.com/elastic/kibana/issues/209065
-describe.skip('Response console', { tags: ['@ess', '@serverless'] }, () => {
+describe('Response console', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     login();
   });
 
-  // Failing: See https://github.com/elastic/kibana/issues/276337
-  describe.skip('File operations:', () => {
+  describe('File operations:', () => {
     const homeFilePath = Cypress.env('IS_CI') ? '/home/vagrant' : '/home/ubuntu';
 
     const fileContent = 'This is a test file for the get-file command.';

--- a/x-pack/solutions/security/plugins/security_solution/public/management/cypress/support/create_and_enroll_endpoint_host_ci.ts
+++ b/x-pack/solutions/security/plugins/security_solution/public/management/cypress/support/create_and_enroll_endpoint_host_ci.ts
@@ -13,6 +13,7 @@ import { kibanaPackageJson } from '@kbn/repo-info';
 import { isServerlessKibanaFlavor } from '../../../../common/endpoint/utils/kibana_status';
 import { fetchFleetLatestAvailableAgentVersion } from '../../../../common/endpoint/utils/fetch_fleet_version';
 import { isFleetServerRunning } from '../../../../scripts/endpoint/common/fleet_server/fleet_server_services';
+import { PINNED_AGENT_VERSION } from '../../../../scripts/endpoint/common/endpoint_host_services';
 import type { HostVm } from '../../../../scripts/endpoint/common/types';
 import type { BaseVmCreateOptions } from '../../../../scripts/endpoint/common/vm_services';
 import { createVm } from '../../../../scripts/endpoint/common/vm_services';
@@ -76,6 +77,10 @@ export const createAndEnrollEndpointHostCI = async ({
     const isServerless = await isServerlessKibanaFlavor(kbnClient);
     if (isServerless) {
       agentVersion = await fetchFleetLatestAvailableAgentVersion(kbnClient);
+    } else {
+      // Temporary: pin the stateful/ESS agent to a version unaffected by the broken Linux
+      // ransomware global artifact (see PINNED_AGENT_VERSION). Remove once fixed upstream.
+      agentVersion = PINNED_AGENT_VERSION;
     }
   }
 

--- a/x-pack/solutions/security/plugins/security_solution/scripts/endpoint/common/endpoint_host_services.ts
+++ b/x-pack/solutions/security/plugins/security_solution/scripts/endpoint/common/endpoint_host_services.ts
@@ -45,6 +45,20 @@ export interface CreateAndEnrollEndpointHostResponse {
 /**
  * Creates a new virtual machine (host) and enrolls that with Fleet
  */
+/**
+ * Temporary pin for the Elastic Agent version used by real-endpoint tests (Cypress + CLI runner).
+ *
+ * The stack (`9.5.0-SNAPSHOT`) Elastic Defend build fails to load the Linux `production-ransomware-v1`
+ * global artifact shipped in the current daily manifest (`ransom-protect` v0.0.3), and treats that
+ * single bad artifact as fatal to the entire global manifest — leaving the endpoint with no
+ * protections configured and the agent permanently `DEGRADED`, so policy status never reaches
+ * "Success". `9.4.3` tolerates the same bad artifact (applies the rest), so it enrolls healthy.
+ *
+ * Remove this pin (let the version fall back to the stack version) once the upstream ransomware
+ * artifact is fixed/rolled back. See https://github.com/elastic/kibana/issues/218441
+ */
+export const PINNED_AGENT_VERSION = '9.4.3';
+
 export const createAndEnrollEndpointHost = async ({
   kbnClient,
   log: _log,
@@ -65,6 +79,10 @@ export const createAndEnrollEndpointHost = async ({
     const isServerless = await isServerlessKibanaFlavor(kbnClient);
     if (isServerless) {
       agentVersion = await fetchFleetLatestAvailableAgentVersion(kbnClient);
+    } else {
+      // Temporary: pin the stateful/ESS agent to a version unaffected by the broken Linux
+      // ransomware global artifact (see PINNED_AGENT_VERSION). Remove once fixed upstream.
+      agentVersion = PINNED_AGENT_VERSION;
     }
   }
   const activeSpaceId = (await fetchActiveSpace(kbnClient)).id;

